service-registry/src/main/java/org/apache/servicecomb/serviceregistry/client/http/WebsocketUtils.java

```diff
@@ -52,18 +53,19 @@ public static void open(IpPort ipPort, String url, Handler<Void> onOpen, Handler
           onConnectFailed.handle(asyncResult.cause());
         } else {
           onOpen.handle(null);
-          asyncResult.result().exceptionHandler(v -> {
+          WebSocket webSocket = asyncResult.result();
```

Comment:
This PR seems nothing chaned. Can you describe when will fail of old code?

Reply:
Yes, just some refactoring. I found that the process of handling this problem is simply refactored, so there is no synchronization process here.

Reply:
I close the issue. code from master is same.